common/test/java/com/couchbase/lite/ReplicatorOfflineTest.java

```diff
@@ -105,6 +106,7 @@ public void testChangeListenerEmptyArg() throws Exception {
     }
 
     @Test
+    @Ignore("Platform no longer has control of network reachablity")
```

Comment:
Platform is still responsible for controlling network reachability. Platform needs to watch for the reachability when the replicator is OFFLINE and inform the C4Replicator to retry when the host is reachable.

Reply:
As noted in the description of this ticket, CBL-760

I though you all were complaining that my commits were to big.  Here's one divided into two, reasonable steps...